app/ante/ante.go

```diff
@@ -49,6 +49,7 @@ func NewAnteHandler(ak auth.AccountKeeper, evmKeeper EVMKeeper, sk types.SupplyK
 				authante.NewSigGasConsumeDecorator(ak, sigGasConsumer),
 				authante.NewSigVerificationDecorator(ak),
 				authante.NewIncrementSequenceDecorator(ak), // innermost AnteDecorator
+				NewValidateMsgHandlerDecorator(),
 			)
 
 		case evmtypes.MsgEthereumTx:
```

Comment:
you can move this to be the top case and then remove the `MsgEthereumTx` on the ValidateMsgHandlerDecorator

Reply:
ok